### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by schnogz**
This pull request was initiated by Lokalise (user Amadeo) at 2021-03-11 18:15:03

### CODE REVIEW

**Comment by amadeo-bc** [comment]
@schnogz fyi the reason why this removes a bunch of keys from the other languages is because @plondon did a cleanup of the en.json file that I've just replicated across to the rest after fixing the lokalise integration.

**Comment by schnogz** [comment]
@plondon I guess i will wait until you approve this since you did the cleanup

**Review by plondon — APPROVED** [review]

**Pull request merged**
